compiler/transf.nim

```diff
@@ -967,20 +968,22 @@ template liftDefer(c, root) =
   if c.deferDetected:
     liftDeferAux(root)
 
-proc transformBody*(module: PSym, n: PNode, prc: PSym): PNode =
-  if nfTransf in n.flags or prc.kind in {skTemplate}:
-    result = n
-  else:
-    var c = openTransf(module, "")
-    result = liftLambdas(prc, n, c.tooEarly)
-    #result = n
+proc transformBody*(ctx: PContext, n: PNode, prc: PSym): PNode =
```

Comment:
This does not take a PContext.

Reply:
@Araq, why not? All invocations of `transformBody` pass `c.module`, so I just changed them to pass `c` :). Do you suggest I should have just added `features` to `transformBody`? I could do it...